### ISSUE TITLE
Support HTTP/3 (and QUIC?) in LibHTTP

### ISSUE THREAD

**Issue by theoparis** [SerenityOS/serenity#24672]
I didn't see any existing issues about this besides DNS-Over-QUIC. I assume this would be a prerequisite for getting http/3 support implemented in Ladybird.

**Comment by DanShaders** [comment]
> getting http/3 support implemented in Ladybird

Not exactly. Ladybird can just yoink some third-party library and call it a day. Supporting (asynchronous version of, as opposed to callback hell version of) HTTP/3 in Serenity Browser / "pure" Ladybird, though, is blocked by quite a lot of things I'm currently (occasionally) working on.
